VBA/Excel-VBA/articles/excel-performance-and-limit-improvements.md

```diff
@@ -1,247 +1,165 @@
+ms.date: 9/28/2017
+---
 
-# Excel Performance: Performance and Limit Improvements
- **Summary:** This article discusses performance improvements in Microsoft Excel 2016 and Microsoft Excel 2010. This article is one of three companion articles about techniques that you can use to improve performance in Excel as you design and create worksheets.For more information about how to improve performance in Excel, see  [Excel Performance: Improving Calculation Performance](excel-improving-calcuation-performance.md) and [Excel Performance: Tips for Optimizing Performance Obstructions](excel-tips-for-optimizing-performance-obstructions.md).
+# Excel performance: Performance and limit improvements
 
-**Applies to:** Excel | Excel 2016 | Excel 2013 | Excel 2010 | Office 2016 | SharePoint Server 2010 | VBA
-
-**In this article**
-
-[Excel 2016 Performance Improvements](#xl2016PerfImp)
-
-[Excel 2010 Performance Improvements](#xl2010PerfImp)
-
-[Conclusion](#office2016excelperf_Conclusion)
-
-[About the Author](#xlAboutAuthors)
-
-[Additional Resources](#office2007excelperf_AdditionalResources)
-
-**Provided by:** ![MVP Contributor](images/mvp_avatar.jpg) Charles Williams,  [Decision Models Limited](http://www.decisionmodels.com/) | [About the Author](excel-improving-calcuation-performance.md#xlAboutAuthor)
-
-
-## Excel 2016 Performance Improvements
+## Excel 2016 performance improvements
 <a name="xl2016PerfImp"> </a>
-Microsoft Office has fundamentally changed its development and release methodology with Office 16 and Office 365. Updates and new features are released on a regular cycle, so it becomes important to note the build number that improvements were released, and to check which build number you are using.The timescale that updates become available to you depends on which update option you are using:
+
+Microsoft Office has fundamentally changed its development and release methodology with Office 2016 and Office 365. Updates and new features are released on a regular cycle, so it becomes important to note the build number that improvements were released, and to check which build number you are using. The timescale that updates become available to you depends on which update option you are using:
 - Insider
 - Monthly Channel
 - Semi-annual Channel
 
-For more details on the Office 2016 release cadence names see [Slow - Fast Level Names](https://support.office.com/en-US/article/Changes-to-Slow-Fast-level-names-for-Office-Insider-for-Windows-desktop-055ee4f9-9ce3-4fb8-8a9a-ca6745867d52).
+For more information about the Office 2016 release cadence names, see [Slow - Fast Level Names](https://support.office.com/en-US/article/Changes-to-Slow-Fast-level-names-for-Office-Insider-for-Windows-desktop-055ee4f9-9ce3-4fb8-8a9a-ca6745867d52).
 
 The following sections discuss some of the features that have been introduced in Excel 2016 that you can use to improve performance with large or complex workbooks.
```

Comment:
I think that this should be the introduction to the article, recast as "Excel 2016 introduces new features that you can use to improve performance when you're working with large or complex Excel workbooks."